+Base/pref.m

```diff
@@ -96,11 +113,35 @@
             % pref_handler. It is intended to give the pref an opportunity
             % to bind a method of the module_instance.
             % Can also use to check/verify input/output
+            obj.parent_class = class(module_instance);
+            
             mc = metaclass(module_instance);
+            
+            props = mc.PropertyList;
+            
+            for i = 1:length(props)
+                if strcmp(props(i).Name, obj.property_name)
+                    obj.read_fn =       @()(module_instance.readProp(props(i)));
+                    obj.writ_fn =       @(val)(module_instance.writProp(props(i), val));
+                    obj.listen_fn =     @(event, callback)(module_instance.addlistener(obj.property_name, event, callback));
+                end
+            end
+            
             methods = mc.MethodList;
+            
             avail_methods = {'set','custom_validate','custom_clean'};
             argouts = [1,0,1];
+            
             for j = 1:length(avail_methods)
+                if j == 1 && isempty(obj.(avail_methods{j}))
+                    fnstring = [avail_methods{j} '_' obj.property_name];
+                    mmethod = methods(strcmp(fnstring,{methods.Name}));
+                    
+                    if ~isempty(mmethod)
+                        obj.(avail_methods{j}) = fnstring;
+                    end
+                end
```

Comment:
To automatically find module.set.prop methods which have been renamed to module.set_prop; this only occurs when the pref.set is empty.